### PR TITLE
feat: add scroll area component

### PR DESCRIPTION
**Pull request by jose-costa-frontify**
CU-8695khnex

being unaware of any specific designs for scrollbars, I have used a slightly modified version of the OG radix styles

### CODE REVIEW

File: packages/components/src/components/ScrollArea/ScrollArea.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { type Meta, type StoryObj } from '@storybook/react';
+
+import { ScrollArea } from './ScrollArea';
+
+type Story = StoryObj<typeof ScrollArea>;
+const meta: Meta<typeof ScrollArea> = {
+    component: ScrollArea,
```
Comment (SamuelAlev) [inline packages/components/src/components/ScrollArea/ScrollArea.stories.tsx:9]:
```suggestion
const meta: Meta<typeof ScrollArea> = {
    title: 'Components/Scroll Area',
    component: ScrollArea,
```

File: packages/components/src/components/ScrollArea/ScrollArea.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    args: {},
+};
+export default meta;
+
+export const Default: Story = {
+    decorators: [
+        (Story) => {
+            return <Story />;
+        },
+    ],
```
[the remark this answers is not in the document]
Comment (jose-costa-frontify) [reply packages/components/src/components/ScrollArea/ScrollArea.stories.tsx:26]: copy pasta leftover

File: packages/components/src/components/ScrollArea/styles/scrollArea.module.scss [hunk 3]
```diff
@@ -0,0 +1,60 @@
+@import '../../../utilities/transitions.module.scss';
```
[the remark this answers is not in the document]
Comment (jose-costa-frontify) [reply packages/components/src/components/ScrollArea/styles/scrollArea.module.scss:1]: the scss files always get me

File: packages/components/src/components/ScrollArea/ScrollArea.stories.tsx
[the remark this answers is not in the document]
Comment (jose-costa-frontify) [reply packages/components/src/components/ScrollArea/ScrollArea.stories.tsx:1]: it does be more better

File: packages/components/src/components/ScrollArea/ScrollArea.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+                    purus. Integer auctor, libero in ultricies tincidunt, odio libero posuere justo, vel ultricies nunc
+                    justo vitae purus. Integer auctor, libero in ultricies tincidunt, odio libero posuere justo, vel
+                    ultricies nunc justo vitae purus.
+                </p>
+            </ScrollArea>
+        );
+    },
+};
+
+export const TwoDimentions: Story = {
```
Comment (SamuelAlev) [inline packages/components/src/components/ScrollArea/ScrollArea.stories.tsx:69]:
```suggestion
export const TwoDimentsions: Story = {
```

File: packages/components/src/components/ScrollArea/__tests__/ScrollArea.ct.tsx [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+            <ScrollArea type="always" maxHeight="300px" maxWidth="300px">
+                <div style={{ height: '1000px', width: '1000px' }}>Scrollable content</div>
+            </ScrollArea>,
+        );
+
+        const verticalScrollbar = component.getByTestId('fondue-scroll-area-vertical-scrollbar');
+        const horizontalScrollbar = component.getByTestId('fondue-scroll-area-horizontal-scrollbar');
+
+        await component.hover();
+        // todo: scroll content
```
Comment (SamuelAlev) [inline packages/components/src/components/ScrollArea/__tests__/ScrollArea.ct.tsx:88]: TODO

File: packages/components/src/components/ScrollArea/styles/scrollArea.module.scss [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    width: 100%;
+    height: 100%;
+    padding-right: var(--scrollbar-size);
+}
+
+.scrollbar {
+    @include transition-colors;
+    display: flex;
+    /* ensures no selection */
+    user-select: none;
```
Comment (SamuelAlev) [inline packages/components/src/components/ScrollArea/styles/scrollArea.module.scss:22]:
```suggestion
    user-select: none;
```

**Review by SamuelAlev — APPROVED** [review]
Approved but small nit comment

**Pull request merged**